paths/electron/starting-with-electron/04-creating-app-files.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Creating App Files
+permalink: /electron/starting-with-electron/create-app-files
+next-page: /electron/starting-with-electron/running-the-app
+facilitator: false
+sidebar:
+  nav: "electron-getting-starting"
+main-content: |
+
+  Since we used a boilerplate, the main files are already there. These files are the
+  - `package.json`
+  - `src/index.html`
+  - `src/index.js`
+
+  We're going to be using other css and Javascript for our project, so let's add those files now.
+  1. Create a new file in the `src` directory named `style.css`.
+  1. Create a new file in the `src` directory named `script.js`.
```

Comment:
Is it possible to run the app at this point, even if it's boring? When I'm learning things, I always hate getting 10 steps in before realizing something doesn't run, and I can never pinpoint if it's because I made a mistake in the setup or in my code.